src/ts/utils/dom.ts

```diff
@@ -19,6 +19,22 @@ export function getActiveEditElement(): ValueElement {
     return element as ValueElement;
 }
 
+export function getTopLevelBlocks() {
+    return document.querySelector('.roam-article div .flex-v-box') as HTMLElement;
+}
+
+export function getLastTopLevelBlock() {
+  const lastChild = getTopLevelBlocks().lastChild as HTMLElement;
+    console.log('bottom-block: ', lastChild);
```

Comment:
something fishy with indentation here and in the next function. Also is logging left intentionally?

Reply:
forgot em